samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs

```diff
@@ -17,13 +17,12 @@ namespace Microsoft.BotBuilderSamples.Dialog
     public class QnAMakerBaseDialog : QnAMakerDialog
     {
         // Dialog Options parameters
-        public const string DefaultNoAnswer = "No QnAMaker answers found.";
         public const string DefaultCardTitle = "Did you mean:";
         public const string DefaultCardNoMatchText = "None of the above.";
         public const string DefaultCardNoMatchResponse = "Thanks for the feedback.";
 
         private readonly IBotServices _services;
-        private readonly IConfiguration _configuration;
+        private readonly string DefaultAnswer = "No QnAMaker answers found.";
```

Comment:
Why set this here? You have a configuration option for this purpose.

Reply:
This is fallback string when configuration entry is empty.